copter/source/docs/thrust_loss_yaw_imbalance.rst

```diff
@@ -0,0 +1,47 @@
+
+These warnings are designed to detect hardware failures in the propulsion system, if they begin to trip on a vehicle that has been flying without warnings the propulsion system should be checked.
+Warning are more likely to happen with larger payloads and in strong/gusty winds.
+
+Potential thrust Loss
+=====================
+
+If potential thrust loss messages are seen on the GCS or in the data-flash logs investigation should be done to find the cause and remedy. The warning will give a motor number, for example:
+
+```Potential Thrust Loss (3)```
+
+These warnings are the result of a motor or motors saturating at 100% throttle, because of this saturation ArduCopter can no longer achieve the requested roll, pitch, yaw and throttle output.
+If this continues for a prolonged period of time the vehicle will have reduced reduced altitude and attitude control and may crash.
+
+If these messages are seen in hover or relaxed flight the problem must be fixed in hardware. The vehicles thrust to weight ratio should be increased, either by a change in propulsion or reduction in mass.
+
+If only seen in climbing and aggressive manuevers it may be sufficiency to lower the requested accelerations and speeds. Again the vehicle thrust to weight ratio can be increased to allow higher accelerations and speeds.
+
+Yaw imbalance
+=============
+
+The yaw imbalance warning is a measure of how hard the vehicle is working to yaw, the warning will trigger before the yaw output is saturated. If the yaw output becomes saturated the vehicles
+ability to maintain yaw will be compromised, in the worst case this will result in the vehicle rapidly spinning. The warning message gives a percentage of maximum yaw output, at 100% it is saturated, for example:
+
+```Yaw Imbalance 87%```
+
+If seen in hover the issue should be resolved in hardware, if the value is increasing with no pilot yaw input the vehicle should be landed immediately. Yaw imbalance can be visualized in dataflash logs by comparing the PWM outputs to the arming_the_motors
+it will present as a large throttle level difference between the clockwise and counter clockwise motors, as shown below:
+
+.. image:: ../../../images/yaw_imbalance_log.png
+    :target: ../_images/yaw_imbalance_log.png
+
+This should be fixed in hardware, the most common cause is the motors not being vertical on a circular arm. If the imbalance still persists
+the motors can be rotated slightly such that the thrust assist the yaw for its rotation direction. Some vehicles can be extremely sensitive to motors thrust vector.
+
+If the warning is only seen in agrsive yaw maneuvers the threshold can be increased by raising :ref:`ATC_RAT_YAW_IMAX<ATC_RAT_YAW_IMAX__AC_AttitudeControl_Multi>` however it may also be worth revisiting the yaw tune.
```

Comment:
```suggestion
If the warning is only seen in aggressive yaw maneuvers, the warning threshold can be increased by raising :ref:`ATC_RAT_YAW_IMAX<ATC_RAT_YAW_IMAX__AC_AttitudeControl_Multi>`.  However, it may also be worth revisiting the yaw tune.
```